### DISCUSSION TITLE
Deeplink from a push message to a hash link for popup

### DISCUSSION THREAD

**Discussion by Predjee** [Clooos/Bubble-Card#910] [Feature requests]
Maybe it's available already but i can't seem to figure it out. What I'm missing is for example the following situation; I'm having motion detected in my camera from the front door. This will send me a push message with an image and a deeplink to the dashboard but also containing the # link for the popup i want to open. If i navigate from the browser it works. But from the push message it doesn't work. Is this a feature which is maybe useful or am i missing something?

**Comment by Clooos** [comment]
Hi, indeed you are not the first to say that, an issue was already opened for that specific use case. I still need to try this to see how I can make it works.
